packages/notification/theme/lumo/vaadin-notification-styles.js

```diff
@@ -132,6 +132,22 @@ registerStyles(
       animation: lumo-notification-exit-slide-down 300ms;
     }
 
+    :host([theme='success']) ::slotted(vaadin-button[focus-ring]) {
```

Comment:
`::slotted` only applies if the child `<vaadin-button>` is a direct child of the notification card. If the button is wrapped inside a `<div>` for example, the style doesn't apply.

I guess we could use CSS variables to fix this